### PR TITLE
Poshi 145 Add method to check for duplicate function and macro names

### PR DESCRIPTION
**Pull request by CalumR23**
https://issues.liferay.com/browse/POSHI-145

### CODE REVIEW

**Comment by CalumR23** [comment]
ci:test:sf

File: modules/test/poshi/poshi-core/src/main/java/com/liferay/poshi/core/PoshiContext.java [hunk 1]
```diff
@@ -604,6 +604,21 @@ private static void _executePoshiFileCallables(
 		}
 	}
 
+	private static void _findDuplicateName(String macroName, String filePath) {
```
Comment (kenjiheigel) [inline modules/test/poshi/poshi-core/src/main/java/com/liferay/poshi/core/PoshiContext.java:607]: change to `_validateUniqueCommandName`

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:forward

**Pull request closed without merging**